android/beagle/src/main/java/br/com/zup/beagle/android/context/operations/grammar/GrammarChars.kt

```diff
@@ -0,0 +1,15 @@
+package br.com.zup.beagle.android.context.operations.grammar
+
+internal object GrammarChars {
+    const val OPEN_PARENTHESES = '\u0028'
```

Comment:
It's better to use the character instead of unicode.
Example:
```suggestion
    const val OPEN_PARENTHESES = '('
```